advocacy_docs/partner_docs/CohesityDataProtectforPostgreSQL/05-UsingCohesityDataProtectforPostgreSQL.mdx

```diff
@@ -197,7 +209,6 @@ as13backup-# ;
 
 ```
 
-
-
-!!! Note After a restore, Cohesity restarts the database automatically using a “sudo, runuser, -l, enterprisedb, -c, /usr/edb/as(version)/bin/pg_ctl start -D /var/lib/edb/as(version)/data” command instead of starting the database with systemctl. This does not allow the server to be tracked by systemd.  If you want to track the server using systemd, you will need to stop and start the database. Go to your server and use ./pg_ctl -D /var/lib/edb/as(version)/ stop and then login as the root user and issue the systemctl edb-as-(version) start command to allow your server to be tracked by systemd.
+!!! Note 
+After a restore, Cohesity restarts the database using a `sudo, runuser, -l, enterprisedb, -c, /usr/edb/as(version)/bin/pg_ctl start -D /var/lib/edb/as(version)/data` command instead of starting the database with systemctl. This doesn't allow the server to be tracked by systemd. If you want to track the server using systemd, you need to stop and start the database. Go to your server and use `./pg_ctl -D /var/lib/edb/as(version)/ stop`. Then, to allow your server to be tracked by systemd, log in as the root user and issue the `systemctl edb-as-(version) start` command.
```

Comment:
```suggestion
After a restore, Cohesity restarts the database using a `sudo, runuser, -l, enterprisedb, -c, /usr/edb/as(version)/bin/pg_ctl start -D /var/lib/edb/as(version)/data` command instead of starting the database with systemctl. This doesn't allow the server to be tracked by systemd. If you want to track the server using systemd, you need to stop and start the database. Go to your server and use `./pg_ctl -D /var/lib/edb/as(version)/ stop`. Then, to allow your server to be tracked by systemd, log in as the root user, and issue the `systemctl edb-as-(version) start` command.
```